src/explore-education-statistics-common/src/services/__tests__/dataBlockService.test.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable prefer-destructuring */
+import dataBlockService, {
+  DataBlockGeoJSON,
+  DataBlockRequest,
+  GeographicLevel,
+} from '@common/services/dataBlockService';
+import testData from '@common/modules/find-statistics/components/charts/__tests__/__data__/testBlockData';
+
+import { dataApi } from '@common/services/api';
+
+jest.mock('@common/services/api', () => {
+  return {
+    dataApi: {
+      post: jest.fn(),
+    },
+  };
+});
+
+describe('dataBlockService', () => {
+  beforeEach(() => {
+    // @ts-ignore
+    dataApi.post.mockImplementation(() => {
+      return Promise.resolve(testData.response);
+    });
+  });
+
+  const dataBlockRequest: DataBlockRequest = {
+    subjectId: 1,
+    geographicLevel: GeographicLevel.National,
+    startYear: '2014',
+    endYear: '2015',
+    filters: ['1', '2'],
+    indicators: ['23', '26', '28'],
+  };
+
+  test('Call datablock service', async () => {
+    const result = await dataBlockService.getDataBlockForSubject(
+      dataBlockRequest,
+    );
+
+    expect(dataApi.post).toBeCalledWith('/Data', dataBlockRequest);
+
+    expect(result).toMatchSnapshot(result);
+  });
+
+  test('DataBlock service returns geojson data', async () => {
```

Comment:
No need to repeat ourselves. Would suggest something like `getDataBlockForSubject returns geojson data`